tasties_app/tests/test_Comment.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from tasties_app.models import Comment, Recipe, Category
+from django.contrib.auth.models import User
+from django.utils import timezone
+from datetime import date
+
+
+@pytest.fixture
+def generate_data():
+    """This fixture create the data we need for tests
+    and save it in the data base. """
+    user3 = User.objects.create_user(username='user3', password='password')
+    test_data = ['Test Recipe1', user3, 'Test Description1', 'Test Directions1', 1, 'recipe.jpeg']
+    category1 = Category.objects.create(category_name="1")
+    recipe1 = Recipe(title=test_data[0], author_id=test_data[1], description=test_data[2],
+                     directions=test_data[3], publication_date=timezone.now(), minutes_to_make=test_data[4],
+                     recipe_picture=test_data[5])
+    category1.save()
+    recipe1.save()
+    recipe1.categories.add(category1)
+    recipe1.save()
+    category1.save()
+    date1 = date.today().strftime("%Y-%m-%d")
+    comment = Comment(author_id=user3, recipe_id=recipe1, publication_date=date1, comment_text='test comment')
+    comment2 = Comment(author_id=user3, recipe_id=recipe1, publication_date=date1,
+                       comment_text='test comment2')
+    comment.save()
+    comment2.save()
+
+
+class TestComment():
+    @pytest.mark.django_db
+    def test_str(self, generate_data):
+        user = User.objects.get(username='user3')
+        comment = Comment.objects.filter(author_id=user)
+        date1 = str(date.today().strftime("%Y-%m-%d"))
```

Comment:
```suggestion
        date = str(date.today().strftime("%Y-%m-%d"))
```
only 1 date. not need more numbering